init/config.js

```diff
@@ -61,4 +61,34 @@ module.exports = {
     calendarId: "c_9c8sc215f8vopmjuj4ogamd7go@group.calendar.google.com",
     discordWebhookUrl: null
   },
+  firstOpportunity: {
```

Comment:
We can make this more flexible by defining opportunities as an array of opportunity.